pages/maindashboard.tsx

```diff
@@ -0,0 +1,45 @@
+import DashboardLayout from '@/components/layouts/DashboardLayout'
+import MCard from '../components/elements/MCard'
```

Comment:
Suppose we could rename to file to something more elaborate, e.g. `ModuleCard` abd `VirtualMachineCard`